src/Hook/Storage/HookRedisStorage.php

```
@@ -0,0 +1,171 @@
+     * This helper method encapsulates the common pattern of:
+     * 1. Incrementing the execution depth
+     * 2. Checking if depth limit is exceeded and logging a warning
+     * 3. Executing the operation (even if depth is exceeded, for graceful degradation)
+     * 4. Decrementing the depth in a finally block
+     *
+     * @template T
+     * @param string $operation Operation name for logging (e.g., 'get', 'set', 'delete')
+     * @param callable(): T $fn The operation to execute
+     * @return T The result of the callable
+     */
+    private function executeWithDepthTracking(string $operation, callable $fn)
```

PHPMD (2.15.0)

Avoid variables with short names like $fn. Configured minimum length is 3. (undefined)

(ShortVariable)